cmd/deploy/deploy.go

```diff
@@ -366,6 +367,7 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 			return err
 		}
 	}
+	oktetoLog.SetStage("")
```

Comment:
what if there is no dependency deployment, this will reset the stage from the previous one?

Reply:
It was already clean in line 329